app/code/community/Bolt/Boltpay/Model/Productpage/Cart.php

```diff
@@ -203,6 +206,13 @@ protected function createCart()
 
             $product = $this->getProductById($productId);
 
+            /** @var Mage_CatalogInventory_Model_Stock_Item $stockItem */
+            $stockItem = $product->getStockItem();
+
+            //defer stock validation to order creation
+            $stockItem->setManageStock(0);
+            $stockItem->setUseConfigManageStock(0);
```

Comment:
The above several lines should be documented / marked for removal when this is fixed on Bolt server side.